android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -1654,11 +1655,21 @@ public boolean dispatchKeyEvent(KeyEvent event) {
     }
 
     public void showOverlay() {
-        removeCallbacks(hideRunnable);
-        setStateOverlay(ControlState.ACTIVE.toString());
+
+        if (controlsAutoHideTimeout != null) {
+            removeCallbacks(hideRunnable);
+            setStateOverlay(ControlState.ACTIVE.toString());
```

Comment:
Can this be a problem that I am setting the state to ACTIVE. It can be also INACTIVE.